### DISCUSSION TITLE
Adding extra elements to full screen (specifically the playlist UI)

### DISCUSSION THREAD

**Discussion by mrjackcitrus** [videojs/video.js#7545] [General]
I am using Video JS with the playlist plugin and the playlist UI plugin and all is working great until I go full screen.

Is there a way to somehow "attach" my playlist UI element to the player so the playlist goes full screen too?

Thanks for your help :)

**Comment by gkatsev** [comment]
In all browsers except iPhones, when we go fullscreen, we take the player element `player.el()` fullscreen. This means that anything that's a child of that will be available in fullscreen.
